tests/lib/rook/ses.py

```diff
@@ -33,10 +34,43 @@ def __init__(self, workspace, kubernetes):
 
     def build(self):
         super().build()
-        logger.info('SES based rook does not require building')
+        self.get_rook()
+        if not converter('@bool', settings.UPSTREAM_ROOK.BUILD_ROOK_FROM_GIT):
+            return
+
+        self.get_golang()
+        logger.info("Compiling rook...")
+        execute(
+            command=f"make --directory {self.build_dir} "
```

Comment:
Can some of this build be deduplicated from the upstream method?

Reply:
Good point. Changed it.